networkx/algorithms/tree/operations.py

```diff
@@ -0,0 +1,113 @@
+# operations.py - binary operations on trees
+#
+# Copyright 2015 NetworkX developers.
+#
+# This file is part of NetworkX.
+#
+# NetworkX is distributed under a BSD license; see LICENSE.txt for more
+# information.
+"""Operations on trees."""
+from functools import partial
+from itertools import chain
+
+import networkx as nx
+from networkx.utils import accumulate
+
+__all__ = ['join']
+
+
+def unzip(iterable):
```

Comment:
I agree, I just find it particularly hard to remember what `zip(*iterable)` does :). I'll inline it.